### PR TITLE
update mandb module

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

File: mandb.if [hunk 1]
```diff
@@ -122,14 +122,19 @@ interface(`mandb_manage_cache_content',`
...
 	mandb_run($1, $2)
 
-	# pending
-	# miscfiles_manage_man_cache_content(mandb_t)
+	ifdef(`init_systemd',`
+		gen_require(`
+			class service { start status stop };
+		')
+
+		allow $1 mandb_unit_t:service { start status stop };
```
Comment (pebenito) [inline mandb.if]: I think this needs to go in the direction of using `init_start_stop_service()`

File: mandb.te [hunk 2]
```diff
@@ -37,24 +37,12 @@ dev_search_sysfs(mandb_t)
...
-	optional_policy(`
-		apt_exec(mandb_t)
-		apt_read_db(mandb_t)
-	')
-
-	optional_policy(`
-		dpkg_exec(mandb_t)
-		dpkg_read_db(mandb_t)
-		userdom_dontaudit_search_user_home_dirs(mandb_t)
-	')
```
Comment (pebenito) [inline mandb.te]: @bigon do you haven any comments on these removals?

File: mandb.if (hunk 1 above)
Comment (cgzones) [inline mandb.if]: probably, but `init_start_stop_service()` requires an initrc_script type
btw, there is no systemd unit file matching `/usr/lib/systemd/system/[^/]*man-db.*` on debian sid, so I added this statement just for fashion

File: mandb.te (hunk 2 above)
Comment (bigon) [inline mandb.te]: mmmh

Post install scripts are executed in dpkg_script_t (the maintainer script) now I think and not dpkg (or even apt, higher in the block). @cgzones did you check that too?

**Comment by cgzones** [comment]
closing for now, needs some more work

**Pull request closed without merging**